### ISSUE TITLE
Symfonymailer::setSender() must be of type string, null given

### ISSUE THREAD

**Issue by SkermBE** [verbb/formie#1116]
### Describe the bug

When a form is submitted the needed form notifications are triggered and we receive our emails. But when it fails or you want te re-send the notification this does not work.

`yii\symfonymailer\Message::setSender(): Argument #1 ($address) must be of type string, null given, called in /var/www/html/vendor/verbb/formie/src/controllers/SentNotificationsController.php on line 282`

### Steps to reproduce

1. Use the pencil 'contact' form
2. Fill in the form
3. Receive the email (or dont)
4. Resend the email from the 'Send notification' window
5. Get the error message

### Form settings

Formie 'contact form' stencil or any kind of from used.

### Craft CMS version

Craft CMS 4.2.5.2 (solo)

### Plugin version

2.0.12

### Multi-site?

yes

### Additional context

Tested with both SMTP2GO and SendGrid(2.0.1) mail adapter.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
